Gemfile

```diff
@@ -1,5 +1,5 @@
 source 'https://rubygems.org'
-gem 'omnibus', git: 'git://github.com/datadog/omnibus-ruby.git', branch: 'custom3.2.2-2'
-gem 'omnibus-software', git: 'git://github.com/datadog/omnibus-software.git', branch: 'master'
+gem 'omnibus', :git => 'git://github.com/datadog/omnibus-ruby.git', :branch => 'etienne/ship-licenses-and-sources'
+gem 'omnibus-software', :git => 'git://github.com/datadog/omnibus-software.git', :branch => 'etienne/omnibus-4-migration' 
```

Comment:
Nope, that's just me not being up to date with the newer ruby Syntax :) I'll fix that when we re-switching the branches we use to `master` (which we will do at least for `omnibus-software` I think) :)